product_variant_supplierinfo/models/procurement.py

```diff
@@ -15,13 +15,14 @@ def _get_product_supplier(self, procurement):
         """returns the main supplier of the procurement's product
            given as argument"""
         product = procurement.product_id
-        company_supplier = self.env['product.supplierinfo'].search([
-            '|',
-            '&',
-            ('product_tmpl_id', '=', product.product_tmpl_id.id),
-            ('product_id', '=', False),
-            ('product_id', '=', product.id),
-            ('company_id', '=', procurement.company_id.id),
+        company_supplier = self.env['product.supplierinfo'].search(
```

Comment:
Why do you change this style? Please don't make the diff bigger only for stylistic reasons (squash your commits when restoring this to avoid several diffs).